lighthouse-cli/bin.js

```diff
@@ -107,8 +107,8 @@ async function begin() {
     // copied over to LH.Settings.extraHeaders, which is LH.Crdp.Network.Headers. Force
     // the conversion here, but long term either the CLI flag or the setting should have
     // a different name.
-    // @ts-ignore
-    let extraHeadersStr = /** @type {string} */ (cliFlags.extraHeaders);
+    /** @type {string} */
```

Comment:
there are three or four places where we had unnecessary `@ts-ignore`s

Reply:
THANKS FOR ALL HELP